low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -128,6 +136,39 @@ def get_controllers_info(self):
             data.append(controller_dict)
         return data
 
+    def get_platform_sensors(self):
+        sensor_list = ['temperature', 'current', 'voltage']
+        sensor_data = self.build_encl_platform_sensors_data(sensor_list)
+        return sensor_data
+
+    def build_encl_platform_sensors_data(self, platform_sensors):
+        sensors_data = {}
+        sensors_resp = self.get_realstor_encl_data('sensor-status')
+        if sensors_resp:
```

Comment:
If no valid response seen, log it atleast about inability to supply this health data

Reply:
Done.

Reply:
I dont see any action through any 'else' case still, what is accomplished ?

Reply:
Please see line number 196 below.

```
<!--StartFragment--> 
  for platform_sensor in platform_sensors:            
    if platform_sensor not in sensors_data:                
    sensors_data[platform_sensor] = f'Unable to retrive {platform_sensor} data'
<!--EndFragment-->

```

Reply:
it doesnt look right, for sensor where data is there, it has full structured json, while missing sensor data, that whole entry has one string, its quite inconsistent. Rather let that entry not be there in sensors_data[]

All i was asking here is to log that the ws call failed, so cant retrieve the sensors data at all

Reply:
As discussed, only addressed 

> while missing sensor data, that whole entry has one string, its quite inconsistent. Rather let that entry not be there in sensors_data[]

Reply:
yes.